karton/core/resource.py

```diff
@@ -201,6 +201,19 @@ def __init__(
         )
         self._fd = _fd
 
+    @property
+    def content(self) -> bytes:
+        """
+        Resource content. Reads the file if the file was not read before.
+
+        :return: Content bytes
+        """
+        if self._content is None:
+            assert self._path is not None
```

Comment:
We can either get the content as `content` or from `path`, right? There is no other way

Reply:
We can check additionally in LocalResource constructor if at least one thing (content/path) is provided and raise ValueError instead.